jsk_fetch_robot/jsk_fetch.rosinstall.kinetic

```diff
@@ -0,0 +1,37 @@
+- git:
+    local-name: jsk-ros-pkg/jsk_recognition
+    uri: https://github.com/jsk-ros-pkg/jsk_recognition
+    version: 1.2.2
+- git:
+    local-name: jsk-ros-pkg/jsk_pr2eus
+    uri: https://github.com/jsk-ros-pkg/jsk_pr2eus.git
+    version: 0.3.7
+- git:
+    local-name: FA-I-sensor
+    uri: https://github.com/RoboticMaterials/FA-I-sensor.git
+- git:
+    local-name: ros-drivers/audio_common
+    uri: https://github.com/furushchev/audio_common.git
+    version: device
+- git:
+    local-name: ros-perception/image_pipeline
+    uri: https://github.com/ros-perception/image_pipeline
+    version: indigo
+- git:
+    local-name: ros/nodelet_core
+    uri: https://github.com/ros/nodelet_core
+    version: 1.9.6
+- git:
+    local-name: strands-project/mongodb_store
+    uri: https://github.com/strands-project/mongodb_store.git
+    version: kinetic-devel
```

Comment:
if we need latest code than version 0.3.7, please add comment on "we're waiting for 0.3.8" and link to the issue URL for requesting release.
https://github.com/strands-project/mongodb_store/commits/kinetic-devel